### DISCUSSION TITLE
Next/Script + Google One Tap

### DISCUSSION THREAD

**Discussion by luyizhang** [vercel/next.js#38941] [Help]
I'm adding google one tap to my website, and am having some trouble getting next/script to work with src `https://accounts.google.com/gsi/client`

It works if I use normal script: `<script src="https://accounts.google.com/gsi/client". async defer />`.

I've tried using strategy lazyOnload, beforeInteractive, and afterInteractive.

**Comment by valcosmos** [comment]
Hello, I seem to have seen a similar question before, u can try this.

```tsx
import "../styles/globals.css";
import type { AppProps } from "next/app";
import Script from "next/script";

function MyApp({ Component, pageProps }: AppProps) {
  return (
    <>
      <Script
        async
        defer
        src="https://accounts.google.com/gsi/client"
        type="module"
      />

      <Component {...pageProps} />
    </>
  );
}

export default MyApp;
```

**Comment by jhourlad01** [comment]
This results to CORS error:
`Access to script at 'https://accounts.google.com/gsi/client' from origin 'https://xxxxxxxx.xxxx' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`
